apps/web/app/components/pool-overview.tsx

```diff
@@ -113,26 +149,91 @@ export function PoolOverview({ pool, poolSlug }: PoolOverviewProps) {
       borderRadius="0px"
     >
       <InfoGrid.Cell gridColumn={['span 2', 'span 2', 'auto']} gridRow={['1', '1', 'span 2']}>
-        <PoolCell pool={pool} poolSlug={poolSlug} />
+        <PoolCell pool={pool} poolSlug={poolSlug} textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['1', '1', '2']} gridRow={['2', '2', '1']}>
-        <HistoricalAprCell />
+        <HistoricalAprCell textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['2', '2', '2']} gridRow={['2', '2', '2']}>
-        <LockupPeriodCell />
+        <LockupPeriodCell textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['1', '1', '3']} gridRow={['3', '3', '1']}>
-        <TotalValueLockedCell />
+        <TotalValueLockedCell textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['2', '2', '3']} gridRow={['3', '3', '2']}>
-        <DaysUntilNextCycleCell />
+        <DaysUntilNextCycleCell textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['1', '1', '4']} gridRow={['4', '4', '1']}>
-        <RewardTokenCell />
+        <RewardTokenCell textStyle={textStyle} />
       </InfoGrid.Cell>
       <InfoGrid.Cell gridColumn={['2', '2', '4']} gridRow={['4', '4', '2']}>
-        <MinimumCommitmentCell />
+        <MinimumCommitmentCell textStyle={textStyle} />
       </InfoGrid.Cell>
     </InfoGrid>
   );
 }
+
+// Add a component for the stacking-amount post label
+export function StackingAmountLabel({ textStyle = "label.03", tagName = "h1" }: { textStyle?: string; tagName?: keyof JSX.IntrinsicElements }) {
+  const label = (content.posts as Record<string, any>)["stacking-amount"]?.Title || "Amount";
```

Comment:
this is very messy code

Reply:
How so?

Reply:
Typ we would so something like...

```ts
const label = 'stacking-amount' in content.posts ? content.posts.title : 'Amount'
```

Ideally we always lowercase props like `title` (at least we do in the extension).